src/com/lilithsthrone/controller/eventListeners/tooltips/TooltipInformationEventListener.java

```diff
@@ -732,7 +732,7 @@ public void handleEvent(Event event) {
 						tooltipSB.append(getEmptyBodyPartDiv("Penis", "Unknown!"));
 					} else {
 						if (owner.hasPenis()) {
-							tooltipSB.append(getBodyPartDiv("Penis", owner.getPenisRace(), owner.getPenisCovering(), owner.isPenisBestial()));
+							tooltipSB.append(getBodyPartDiv("Penis", owner.getPenisRace(), owner.getPenisCovering(), owner.isPenisBestial(), owner.getPenisRawSizeValue() + " inches"));
```

Comment:
This should be made to comply with #931, not everyone will be playing the game with inches.

Reply:
Sure thing, here is an updated version
```suggestion
							tooltipSB.append(getBodyPartDiv("Penis", owner.getPenisRace(), owner.getPenisCovering(), owner.isPenisBestial(), "[unit.sizes(" + owner.getPenisRawSizeValue()+ ")]"));
```